charts/argo-cd/README.md

```diff
@@ -218,7 +218,9 @@ NAME: my-release
 | configs.knownHosts.data.ssh_known_hosts | string | See [values.yaml] | Known Hosts |
 | configs.knownHostsAnnotations | object | `{}` | Known Hosts configmap annotations |
 | configs.repositories | object | `{}` | Repositories list to be used by applications |
+| configs.repositoryCredentialSecretAnnotations | object | `{}` | Repository credential secret annotations |
 | configs.repositoryCredentials | object | `{}` | *DEPRECATED:* Instead, use `configs.credentialTemplates` and/or `configs.repositories` |
```

Comment:
`repositoryCredentialSecretAnnotations` follows the deprecated way to configure credentials.
Better use `configs.credentialTemplatesAnnotations` or something similar